tests/test-sources/plugins/lsp/schemastore.nix

```diff
@@ -33,7 +36,9 @@
         enable = true;
 
         json = {
-          enable = true;
+          # TODO: re-enable when pkg is fixed and available
+          # https://github.com/NixOS/nixpkgs/pull/335559
+          enable = false;
```

Comment:
I, uh, definitely left these warnings for this PR... uh, yeah.

Reply:
Warnings are easily missed in the tests, especially when trusting CI's big green tick.

That's the main motivation for wanting to fail on warnings to begin with :grin:

Maybe we could also have a "ignore all warnings" option too though, which might be better for these kinda temporary disablements?

Reply:
I was fine with that being in here, just making a joke about my temporary disables in the flake.lock update leaving warnings. :P